manifests/facts.pp

```diff
@@ -4,6 +4,7 @@
   String $configdir = $mcollective::configdir,
   String $factspath = $mcollective::factspath,
   Integer $refresh_interval = $mcollective::facts_refresh_interval,
+  String[1] $runmode = $mcollective::periodic_runmode,
```

Comment:
better if this was an Enum of the valid values so it helps people figure out what to put there

Reply:
For consistency with init.pp:

```suggestion
  Enum["cron", "systemd.timer"] $runmode = $mcollective::periodic_runmode,
```

But on second though, there is no need to have a "local" parameter here: we can use the value from `$mcollective::periodic_runmode` directly below.